schemas/telemetry/frecency-update/frecency-update.1.schema.json

```diff
@@ -25,12 +25,15 @@
           "minimum": 0,
           "type": "number"
         },
+        "study_variation": {
+          "type": "string"
+        },
         "update": {
           "items": {
             "type": "number"
           },
-          "maxItems": 18,
-          "minItems": 18,
+          "maxItems": 22,
+          "minItems": 22,
```

Comment:
If there are any messages still being sent with the current iteration of the schema, this change (and the one making the new `study_variation` field required) will cause them to fail validation - is that the desired effect?

Reply:
All the messages sent so far were just from me debugging and testing my addon, so we should probably delete those messages anyways. All the new pings I'm sending will follow the new format.